moped-editor/src/views/projects/projectsListView/components/SaveUserViewModal.js

```diff
@@ -13,17 +13,32 @@ const SaveUserViewModal = ({
   filtersLabels,
   setIsViewSaved,
   handleSnackbar,
+  searchTerm,
 }) => {
   const [saveView, { loading }] = useMutation(ADD_USER_SAVED_VIEW);
 
   let { pathname, search } = useLocation();
 
-  const defaultDescription = filtersLabels
-    .map(
-      (filter) =>
-        `${filter.filterLabel} ${filter.operatorLabel} ${filter.filterValue}`
-    )
-    .join(", ");
+  const getDefaultDescription = () => {
```

Comment:
sets the default description based on what the user has selected

Reply:
@tillyw I think this could be a good place to use `useMemo`.

Maybe something like:
```js
const defaultDescription = useMemo(() => {
// all the same default name prep in your function
}, [the dependencies here])
```